### ISSUE TITLE
WindowVirtualizer is not properly work with static exported nextjs app: "Uncaught TypeError: X is not a function

### ISSUE THREAD

**Issue by Monstass** [inokawa/virtua#322]
WindowVirtualizer does not work properly with static exported nextjs app. 
Rendered items are not clickable and in console we can see error "Uncaught TypeError: X is not a function".

**To Reproduce**
Build and start nextjs app by using 'output: "export"' flag in next config.
```js
import { WindowVirtualizer } from "virtua";

const MainPage = () => {
  const length = 200;

  return (
    <WindowVirtualizer>
      {Array.from({ length: length }).map((_, i) => (
        <div key={i}>
          <a href="http://www.google.com">{`index - ${i}`}</a>
        </div>
      ))}
    </WindowVirtualizer>
  );
};

export default MainPage;
```

Clicking on links has no effect (other events neither) after scroll down for few pixels.
Console output will be like this on every scroll event: 

![image](https://github.com/inokawa/virtua/assets/63319456/c7ec2fc3-46cd-4918-8073-a5cb948b20a1)

Codesandbox with exmaple: https://codesandbox.io/p/devbox/virtua-windowvirtualizer-bug-hj8w6c

**Expected behavior**
No uncaught errors and clickable items.

**Platform:**
- virtua: 0.20.3
- next: 13.5.4
- react: 18

**Additional context**
And, yeah, rolling back to v0.17.6 helped me.

**Comment by inokawa** [comment]
Could you make a reproduction code or give me more detailed information?

**Comment by Monstass** [comment]
@inokawa of course I just accidentally pressed the enter button, my bad :D already updated

**Comment by inokawa** [comment]
Thank you for the information!
It can be reproduced. And  it looks like a SWC's minify issue (like https://github.com/vercel/next.js/issues/58959 ?) because setting `swcMinify: false` to nextconfig solved the error.
```ts
const nextConfig = {
  swcMinify: false,
  output: "export",
};
```

**Comment by Monstass** [comment]
> setting swcMinify: false to nextconfig to use terser solved the error.

Yeah, it is. Thank you for tip.

By the way, without setting ```swcMinify: false``` on 14.0.4 and canary next releases the application crashes completely, but, again, not with ```virtua@0.17.6```

**Comment by inokawa** [comment]
Probably fixed in `0.20.4`.
